### PR TITLE
Added instructions in README.md to build the utility on Ubuntu

### PR DESCRIPTION
**Pull request by yurida**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,24 @@ You can do it on fedora with next comand:
...
+sudo apt install -y libftdi-dev libusb-dev libusb-1.0-0-dev libftdi1-dev
+```
+
 After that simply run make:
 
 ```
 make
 ```
 
+Example of running the utility on host where connected only one EvB:
```
Comment (abrodkin) [inline README.md]: I'd prefer to use more formal wording, i.e. "EvB" -> "development board".

File: README.md [hunk 2]
```diff
@@ -19,12 +19,24 @@ You can do it on fedora with next comand:
...
+
+```
+sudo ./rff-reset --only-one
```
Comment (abrodkin) [inline README.md]: With right privileges set there's no need to use `sudo`, so let's not add confusion here and remove `sudo`. Remember not every user (especially in big corporate environments) may have rights to execute `sudo` ;)

**Review by abrodkin — CHANGES_REQUESTED** [review]
Thanks for the contribution. See my notes though.

File: README.md (hunk 1 above)
Comment (yurida) [reply README.md]: renamed.

File: README.md (hunk 2 above)
Comment (yurida) [reply README.md]: Agree. "sudo" removed.

**Review by abrodkin — APPROVED** [review]

**Comment by abrodkin** [comment]
@yurida thanks a lot for your contribution!

**Pull request merged**